File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int curr = 0;
        for (int next = 1; next < nums.Length; ++next)
        {
            if (nums[curr] != nums[next])
            {
               curr++;
               nums[curr] = nums[next];
            }
        }
        return curr + 1;
    }
}

File: Data Structures & Algorithms/merge-two-sorted-linked-lists/submission-1.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:
    def mergeTwoLists(self, list1: Optional[ListNode], list2: Optional[ListNode]) -> Optional[ListNode]:
        l1 = list1
        l2 = list2

        nl = ListNode()
        head = nl
        while l1 and l2:
            if l1.val < l2.val:
                nl.next = l1
                l1 = l1.next
            else:
                nl.next = l2
                l2 = l2.next
            nl = nl.next
        tmp = l1 if l1 else l2
        if tmp:
            while tmp:
                nl.next = tmp
                nl = nl.next
                tmp = tmp.next
        return head.next